lib/requester/core.js

```diff
@@ -47,14 +47,22 @@ var _ = require('lodash'),
      * @param {Boolean} param.disabled - A flag that can be set to true to indicate the disabled status of the param.
      * @param {String} param.key - The name of the current body parameter.
      * @param {String} param.value - The value of the current body parameter.
+     * @param {String} param.contentType - The media type of the current body parameter.
```

Comment:
mime type?

Reply:
Yes, [formerly known as MIME types](https://www.iana.org/assignments/media-types/media-types.xhtml) :sweat_smile:

Reply:
@codenirvana Are we adding a `contentType` property to body row?

Have we accounted for changes to Schema and postman-collection-transformer in that case?

Reply:
@kamalaknn [Yes](https://github.com/postmanlabs/postman-runtime/pull/623/files#diff-16cbfd6625366bffab729290792ee335).

The schema is updated and no additional changes are made in the transformer as it will not filter keys in the form param.

Reply:
Have you added tests for the same though?

Reply:
No 🤦‍♂️ , will add tests in the transformer.

Reply:
Thanks 👍